api_bugzilla.py

```diff
@@ -243,7 +340,8 @@ def report_bugzilla_desktop_bugs(self, payload):
                             bugzilla_bug_created_at=row['created_at'],
                             bugzilla_bug_last_change_time=row['last_change_time'], # noqa
                             bugzilla_bug_whiteboard=None if pd.isna(row['whiteboard']) else row['whiteboard'], # noqa
-                            bugzilla_bug_keyword=bugzilla_bug_keyword
+                            bugzilla_bug_keyword=bugzilla_bug_keyword,
+                            bugzilla_bug_resolved_at=None if pd.isna(row['resolved_at']) else row['resolved_at'] # noqa
```

Comment:
Added resolved_at (when a bug is closed/fixed) so that we have the new data point requested